### PR TITLE
Improved chargen pods

### PR DESCRIPTION
**Pull request by PsyCommando**
## Description of changes
Changed deprecated walls for current ones, and re-laid out stuff
* Decorated a bit
* Used non-deprecated placeholder walls, and some new flooring.
* Added a bunch of unsimulated flooring.
* Added indestructible and interaction proof wall type for chargen.

2022-07-27 18_15_28-Nebula 13 - Kleibkhar Colony
2022-07-27 18_15_14-Nebula 13 - Kleibkhar Colony

## Changelog
:cl:
tweak: Made the chargen pods a bit prettier.
/:cl:

### CODE REVIEW

**Comment by PsyCommando** [comment]
Updated look:
2022-07-27 22_32_12-Nebula 13 - Kleibkhar Colony

File: code/game/turfs/unsimulated/floor.dm [hunk 1]
```diff
@@ -19,3 +19,80 @@
```
Comment (NataKilar) [inline code/game/turfs/unsimulated/floor.dm:19]: Shouldn't these turfs be in the persistence modpack? Or do you plan on pushing them upstream

Comment (PsyCommando) [reply code/game/turfs/unsimulated/floor.dm:19]: I could push them upstream yeah

**Comment by PsyCommando** [comment]
Wtf is wrong with mapdiffbot. How the hell is it stuck on a 10x10 map... :/

**Comment by PsyCommando** [comment]
@AffectedArc07 I have no ideas why, but mapdiffbot has been stuck for 10hrs on a 10x10 map? Halp please?

**Comment by AffectedArc07** [comment]
> 

No idea lol.

Just push a new commit that adds a newline or something.

**Comment by PsyCommando** [comment]
> > 
> 
> No idea lol.
> 
> Just push a new commit that adds a newline or something.

oke, thanks!

**Comment by PsyCommando** [comment]
Nice, it worked this time!

**Review by NataKilar — APPROVED** [review]
The addition to the blend turfs still icks me out a little bit, but since it's unlikely to be an issue this is fine.

**Pull request merged**